.storybook/preview.tsx

```diff
@@ -16,7 +17,7 @@ import { getEnL10nBundlesSync } from "../src/app/functions/server/mockL10n";
 const inter = Inter({ subsets: ["latin"], variable: "--font-inter" });
 
 const AppDecorator: Exclude<Preview["decorators"], undefined>[0] = (
-  storyFn
+  storyFn,
```

Comment:
The added trailing spaces are being added by Prettier due to recent changes in our linting rules.